config/routes.rb

```diff
@@ -5,6 +5,11 @@
     resources :courses, only: %i[index show], path: 'pathways' do
       resources :lessons, only: %i[show], path: 'steps'
     end
+    get 'auth/log_in'
+    post 'auth/callback'
+    get 'auth/logout'
```

Comment:
Nitpick: `log_in` + `log_out` OR `login` + `logout` :D

Reply:
Ended up going with `sign_in` and `sign_out` but open to changing those if need be.

I read this book a while back https://www.nngroup.com/books/homepage-usability/ and it recommends Sign In/Out